# Position of the leg end-effector in Cartesian space
float64 x     # X coordinate in mm
float64 y     # Y coordinate in mm
float64 z     # Z coordinate in mm
